README.md

```diff
@@ -63,6 +63,35 @@ This will start the API, wait for it to respond, run the end-to-end tests agains
 These tests are located under the `e2e` Gradle sub-project directory.  Like any Gradle project, there are the `main` and `test` directories.
 The `test` directory contains the tests.  The `main` directory contains our custom framework that helps us interact with the API.
 
+#### Load Testing
+
+Load tests are completed with [Locust.io](https://docs.locust.io/en/stable/installation.html). To run load tests:
+
+Install Python (3.7 or later), then...
+
+```shell
+pip3 install locust
+```
+
+You can quickly validate the install with `locust -V`.
+
+A *locustfile.py* is required to run the tests and there is currently one located
+in `/operations`
+
+Run the application...
```

Comment:
Is this a sub-heading or some sort?  Or is this supposed to be part of a sentence that goes with the above or below paragraphs?

Reply:
'Run the application...' was a step. I think I fixed it in a way that makes more sense. Let me know what you think.